src/core/trulens/core/database/migrations/env.py

```diff
@@ -25,7 +25,7 @@
     fileConfig(config.config_file_name)
 
 # Get `sqlalchemy.url` from the environment.
-if config.get_main_option("sqlalchemy.url", None) in (None, ""):
+if config.get_main_option("sqlalchemy.url", None) is None:
```

Comment:
this change is b/c I've merged in https://github.com/truera/trulens/pull/1355/files so we no longer need to consider the empty string